### PR TITLE
CountsAndLists: Move to Beta Protobuf

### PR DESCRIPTION
**Pull request by Kalaiselvi84**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, press enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind breaking
> /kind bug
> /kind cleanup
> /kind documentation
> /kind feature
> /kind hotfix
> /kind release

**What this PR does / Why we need it**:

**Which issue(s) this PR fixes**:

Work on #3775 

**Special notes for your reviewer**:
This PR moves all the Counterss & Lists functions from Alpha to Beta protobuf and updates the relevant files accordingly.

### CODE REVIEW

File: pkg/sdkserver/localsdk.go [hunk 1]
```diff
@@ -42,8 +42,8 @@ import (
 
 var (
 	_ sdk.SDKServer   = &LocalSDKServer{}
-	_ alpha.SDKServer = &LocalSDKServer{}
-	_ beta.SDKServer  = &LocalSDKServer{}
+	_ alpha.SDKServer = &AlphaSDKAdapter{}
```
[the remark this answers is not in the document]
Comment (Kalaiselvi84) [reply pkg/sdkserver/localsdk.go]: I was trying to differentiate alpha and beta features. I have removed it.

File: cmd/sdk-server/main.go [hunk 2]
```diff
@@ -199,9 +199,12 @@ func registerLocal(grpcServer *grpc.Server, ctlConf config) (func(), error) {
 		return nil, err
 	}
 
+	alphaSDKAdapter := &sdkserver.AlphaSDKAdapter{LocalSDKServer: s}
+	betaSDKAdapter := &sdkserver.BetaSDKAdapter{LocalSDKServer: s}
 	sdk.RegisterSDKServer(grpcServer, s)
-	sdkalpha.RegisterSDKServer(grpcServer, s)
-	sdkbeta.RegisterSDKServer(grpcServer, s)
+	sdkalpha.RegisterSDKServer(grpcServer, alphaSDKAdapter)
```
[the remark this answers is not in the document]
Comment (Kalaiselvi84) [reply cmd/sdk-server/main.go]: reverted this change

File: proto/sdk/alpha/alpha.proto [hunk 3]
```diff
@@ -128,65 +128,6 @@ service SDK {
...
-
-    // Removes a value from a List and returns updated List. Returns NOT_FOUND if the List does not exist.
-    // Returns NOT_FOUND if the value is not in the List.
-    rpc RemoveListValue(RemoveListValueRequest) returns (List) {
-      option (google.api.http) = {
-        post: "/v1alpha1/lists/{name}:removeValue"
-        body: "*"
-      };
-    }
 }
```
[the remark this answers is not in the document]
Comment (Kalaiselvi84) [reply proto/sdk/alpha/alpha.proto:132]: Deleted the CL related lines

**Comment by Kalaiselvi84** [comment]
`pkg/sdk/alpha/*` and `pkg/sdk/beta/*` files are autogenerated, do we need to execute any command to modify these file?

**Review by igooch — COMMENTED** [review]
Looks like the tests are failing `test-gen-all-sdk-grpc`. Run `make gen-all-sdk-grpc` from within the /build directory. Once you run that you'll also need to update the REST test/sdk/restapi/http-api-test.go to use beta.

**Comment by Kalaiselvi84** [comment]
> Looks like the tests are failing `test-gen-all-sdk-grpc`. Run `make gen-all-sdk-grpc` from within the /build directory. Once you run that you'll also need to update the REST test/sdk/restapi/http-api-test.go to use beta.

I ran `make gen-all-sdk-grpc` to modify autogenerated files, updated `build/build-sdk-images/restapi/gen.sh` to automate beta file generation, and adjusted `test/sdk/restapi/http-api-test.go` to change alpha to beta for CL

File: proto/sdk/beta/beta.proto [hunk 4]
```diff
@@ -31,4 +37,185 @@ option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_swagger) = {
...
+// Store a count variable.
+message Count {
+    int64 count = 1;
+}
+
+// Store a boolean result
+message Bool {
+    bool bool = 1 [(grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {format: "boolean"}];
+}
+
```
[the remark this answers is not in the document]
Comment (Kalaiselvi84) [reply proto/sdk/beta/beta.proto]: Removed these lines

File: test/sdk/go/sdk-client-test.go [hunk 5]
```diff
@@ -160,36 +160,36 @@ func testPlayerTracking(sdk *goSdk.SDK) {
 func testCounts(sdk *goSdk.SDK) {
 	// LocalSDKServer starting "rooms": {Count: 1, Capacity: 10}
 	counter := "rooms"
-	count, err := sdk.Alpha().GetCounterCount(counter)
+	count, err := sdk.Beta().GetCounterCount(counter)
```
[the remark this answers is not in the document]
Comment (Kalaiselvi84) [reply test/sdk/go/sdk-client-test.go:163]: modified with beta in test/sdk/csharp/Program.cs

**Comment by Kalaiselvi84** [comment]
I will push a new tag for `simple-game-server` to the registry.

**Review by igooch — APPROVED** [review]
LGTM. @markmandel did you want to take a last look?

We still need to update documentation, which can be done as part of a different PR.

**Review by zmerlynn — APPROVED** [review]

**Comment by zmerlynn** [comment]
I'm going to go ahead and merge this since it's so wide - @markmandel if you wsnt to follow up we still have the rest of the release. :)

Great job, @Kalaiselvi84 and @igooch !

File: sdks/nodejs/lib/alpha/alpha_grpc_pb.js
Comment (steven-supersolid) [inline sdks/nodejs/lib/alpha/alpha_grpc_pb.js:1]: Just for posterity but this removed functions from the file and there is no corresponding addition to a `beta_grpc_pb.js` - in fact that file is not generated yet

**Pull request merged**